### ISSUE TITLE
Improve test coverage to 100%

### ISSUE THREAD

**Issue by jrgriffiniii** [samvera/hydra-works#342]

**Comment by tpendragon** [comment]
@mark-dce Is this reasonable to accomplish, or is the current test coverage sufficient?

**Comment by mark-dce** [comment]
Agreed - the additional 1% coverage doesn't seem worth independent development effort.  I'd be happy if this occurs naturally in the course of other work, but 99% seems really good for now.
